src/Navigation/Search/searchSlice.js

```diff
@@ -21,6 +35,7 @@ const searchSlice = createSlice({
             state.data = data;
```

Comment:
I seams to me that You do not pass any data

Reply:
Hey, are you sure you check the latest changes :P? state.data is no longer in the code

Reply:
I'm looking at latest commits pushed to this PR.